First line of code in Quickstart is broken

Was very exciting to try your software. Started from Quickstart, but literally first line of code example is broken:

```js
const { KratosPublicSDK } = require('@oryd/kratos-client')

or

import { KratosPublicSDK } from '@oryd/kratos-client'
```

gives

```
TypeError: KratosPublicSDK is not a constructor

or

index.ts:1:10 - error TS2305: Module '"./node_modules/@oryd/kratos-client/dist/api"' has no exported member 'KratosPublicSDK'.

1 import { KratosPublicSDK } from '@oryd/kratos-client'
```

I know documentation is hard. But first line of code in introductionary Quickstart is too much 😀Any chance to fix it?

--- a/docs/docs/quickstart.mdx
+++ b/docs/docs/quickstart.mdx
@@ -60,10 +60,16 @@ To ensure that no one can access the dashboard without prior authentication
 ```js
 // Import the ORY Kratos SDK. SDKs are available for all popular programming
 // languages! We will add examples for other programming languages here soon.
-import { KratosPublicSDK } from '@oryd/kratos-client';
+import {
+  AdminApi,
+  PublicApi,
+} from '@oryd/kratos-client'
+
+const publicEndpoint = new PublicApi('https://public.ory-kratos')
+const adminEndpoint = new AdminApi('https://admin.ory-kratos')
 
 const needsLogin = (req, res, next) => {
-  new KratosPublicSDK('https://public.ory-kratos')
+  new publicEndpoint
     .whoami(req)
     .then(({ body }) => {
       req.user = { session: body };

